shell/browser/api/electron_api_app.cc

```diff
@@ -998,13 +998,6 @@ void App::SetPath(gin_helper::ErrorThrower thrower,
   if (key >= 0) {
     succeed =
         base::PathService::OverrideAndCreateIfNeeded(key, path, true, false);
-    if (key == DIR_USER_DATA) {
-      succeed |= base::PathService::OverrideAndCreateIfNeeded(
-          chrome::DIR_USER_DATA, path, true, false);
-      succeed |= base::PathService::Override(
-          chrome::DIR_APP_DICTIONARIES,
```

Comment:
I assume `DIR_APP_DICTIONARIES` now works correctly as it is resolved based on `DIR_USER_DATA`? Can you double check this?

Reply:
yeah, that's the idea. How do I check it?